src/components/main/StaffList.css

```diff
@@ -0,0 +1,8 @@
+.btext{
```

Comment:
This is reiterating a comment from Emma earlier, but clearer variable names help everyone understand better. I recommend naming these plusSign and minusSign. Also, it's not clear these are clickable when you hover over them. You can add a CSS property to both .btext and .btext2 that is 'cursor: pointer'